apps/commudle-admin/src/app/feature-modules/hackathon-control-panel/components/hackathon-control-panel-sponsor/hackathon-control-panel-sponsor.component.ts

```diff
@@ -1,17 +1,22 @@
 import { NbDialogService } from '@commudle/theme';
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { faPlus, faFileImage, faXmark } from '@fortawesome/free-solid-svg-icons';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { HackathonService } from 'apps/commudle-admin/src/app/services/hackathon.service';
 import { IHackathonSponsor, IHackathonSponsorGroupedByTierName } from 'apps/shared-models/hackathon-sponsor';
 import { ToastrService } from '@commudle/shared-services';
+import { Subscription } from 'rxjs';
+import { IHackathon } from '@commudle/shared-models';
+import { ICommunity } from 'apps/shared-models/community.model';
+import { ICommunityGroup } from 'apps/shared-models/community-group.model';
+import { SeoService } from 'apps/shared-services/seo.service';
```

Comment:
let's use @commmudle/shared-models for ICommunity and SeoService from @commudle/shared-service